index.js

```diff
@@ -22,7 +22,6 @@ server.route({
 , path: '/new'
```

Comment:
What do you think about updating the routes in this PR?

`/v1/instances` for example